modyn/supervisor/internal/eval/handler.py

```diff
@@ -15,7 +15,22 @@ class EvalRequest(BaseModel):
     trigger_id: int
     training_id: int
     id_model: int
-    most_recent_model: bool
+    currently_active_model: bool | None = Field(
+        None,
+        description=(
+            "Whether the model is currently used for inference and can therefore be considered the active. "
+            "This means that the evaluation interval starts after the model was trained. "
+            "Only available for `after_pipeline` evaluations"
+        ),
+    )
+    currently_trained_model: bool | None = Field(
+        None,
+        description=(
+            "Whether the interval is in a range whether the model is currently trained on. "
+            "For a fixed interval this is typically the model after the currently active model. "
```

Comment:
Isn't it before the currently active model? First we are trained, for the next interval it's active

Reply:
Your are looking at a mapping from model -> interval. We should take the interval -> model perspective. For one certain interval we have one active model and then we start training another one (but are not finished yet in the current interval). Therefore the `currently_trained_model` has a higher id than the `currently_active_model`. For one fixed model the interval using it as `currently_trained_model` obviously comes before the interval with the model as `currently_active_model`.

Reply:
Sorry, brain fart. Makes complete sense...